scripts/wmt14/prepare_MT_data.py

```diff
@@ -0,0 +1,6 @@
+import sys
```

Comment:
Use lowercase for filenames, as elsewhere.

Reply:
Done.